bitnami/external-dns/values.yaml

```
@@ -14,6 +15,12 @@ global:
   ##   - myRegistryKeySecretName
   ##
   imagePullSecrets: []
+    
+  commonLabels: {}
+  ## E.g.
+  ## commonLabels:
+  ##   team_name: dev
+  ##
```

Could you please also take a look at the position of the example? (You can simply apply this suggestion so you don't to do any change)
```suggestion
  imagePullSecrets: []
  ## E.g.
  ## commonLabels:
  ##   team_name: dev
  ##
  commonLabels: {}
```